src/scripts/main.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const rows = [...tbody.querySelectorAll('tr')];
+
+  const newRow = rows[1].cloneNode(true);
+
+  tbody.appendChild(newRow);
+
+  if ([...tbody.querySelectorAll('tr')].length === max) {
+    setButton(e.target, true);
+  } else {
+    setButton(removeRow, false);
+  }
+});
+
+removeRow.addEventListener('click', (e) => {
+  if (tbody.hasChildNodes()) {
+    tbody.removeChild(tbody.lastElementChild);
+  }
```

Comment:
Instead of checking if the tbody has any child nodes, you can check if the rows array has any elements. This would be more efficient because it avoids querying the DOM.